Frends.MicrosoftDynamics365.Request/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.0.1] - 2024-07-03
```

Comment:
The version bumps should be to minor version, so this would be 1.1.0. Last number increases are for development and testing only during development.